setup.py

```diff
@@ -35,18 +34,17 @@
         'Programming Language :: Python :: 3.7',
         'Topic :: Software Development :: Build Tools'
         'Topic :: Software Development :: Embedded Systems',
-    ),
-    description="Mbed Tools: Target information for building Mbed OS",
-    keywords="Arm Mbed OS MbedOS build target platform board module",
+    ],
+    description='Mbed Tools: Target information for building Mbed OS',
+    keywords='Arm Mbed OS MbedOS build target platform board module',
     include_package_data=True,
     install_requires=requirements,
     license='Apache 2.0',
     long_description_content_type='text/markdown',
     long_description=long_description,
     name=PACKAGE_NAME,
-    package_dir={'': SOURCE_DIR},
-    packages=find_packages(SOURCE_DIR),
+    packages=[SOURCE_DIR],
```

Comment:
what if we have subpackages within 'mbed_targets'?

Reply:
Could you give me an example of such case? Unless we see an immediate need for it, I'd argue applying YAGNI rule for now. ;)

Reply:
any submodule that we want public. There is currently no example of this in mbed-targets but since we are using it as a template, I think it is worth considering that it may happen for other tools

Reply:
Resolving this for now as we took the discussion offline. Would like @madchutney to chip in as well.

I'd like to understand how `packages=` influences what's available. I understand it makes its contents available to import globally, but in our case:
```
find_packages(exclude=('tests',)) == ['mbed_targets', 'mbed_targets._internal']
```
 
What does `mbed_targets._internal` being there mean?